docs/src/main/sphinx/admin/properties.rst

```diff
@@ -21,3 +21,80 @@ may be used to tune Trino or alter its behavior when required.
     Logging <properties-logging>
     Web UI <properties-web-interface>
     Regular expression function <properties-regexp-function>
+
+Property value types
+--------------------
+
+Configuration properties in Trino take values of different types, some standard
+and some non-standard. Supported values may be further limited on a per-property
+basis.
+
+.. _prop-type-boolean:
+
+``boolean``
+^^^^^^^^^^^
+
+Properties of a ``boolean`` type support two values, ``true`` or ``false``
+in lowercase.
+
+.. _prop-type-data-size:
+
+``data size``
+^^^^^^^^^^^^^
+
+Properties of a ``data size`` type support values describing an amount of
+data, measured in byte-based units. For example, the value ``6GB`` describes
+six gigabytes.
+
+The ``data size`` type supports the following units:
+
+* ``B``: Bytes
+* ``kB``: Kilobytes
+* ``MB``: Megabytes
+* ``GB``: Gigabytes
+* ``TB``: Terabytes
+* ``PB``: Petabytes
+
+Some ``data size`` properties also support ``unlimited`` as a value.
+
+.. _prop-type-double:
+
+``double``
+^^^^^^^^^^
+
+Properties of a ``double`` type support numerical values, including decimals.
+
+.. _prop-type-duration:
+
+``duration``
+^^^^^^^^^^^^
+
+Properties of a ``duration`` type support values describing an amount of time.
```

Comment:
Again I'd start with "The properties of... support values that describe..."

Reply:
worth noting that there are no negative durations?